src/users/users.controller.ts

```diff
@@ -225,7 +225,7 @@ export class UsersController {
         },
       });
       res.type('image/png');
-      return readable;
+      return readable.read();
```

Comment:
Simply return `user.photo` instead of creating a stream only to immediately read it.

```suggestion
      return user.photo;
```